packages/pwa-kit-react-sdk/bin/upload.js

```diff
@@ -62,10 +62,19 @@ try {
     Utils.fail(message)
 }
 
+// Set the deployment target env var, this is required to ensure we
+// get the correct configuration oject.
+process.env.DEPLOY_TARGET = argv.t
+
+// Get the config object for the current target, if no target was set
+// it will return the defualt configuration and that one will be
+// validated.
+const config = Utils.getConfig()
```

Comment:
Shall we remove the code in this file where it tries to read package.json and grabs value from there?

I'm talking about the code [here](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/packages/pwa-kit-react-sdk/bin/upload.js#L49-L63).

Reply:
The `package.json` file is still being used for project slug is one isn't provided via the command line.